Desserts/Views/MacawChartView.swift

```diff
@@ -0,0 +1,135 @@
+//
+//  MacawChartView.swift
+//  Desserts
+//
+//  Created by 김예랑 on 2020/09/18.
+//  Copyright © 2020 김예랑. All rights reserved.
+//
+
+import Foundation
+import Macaw
+
+class MacawChartView: MacawView {
+    
+    static let thisWeek = createDummyData()
+    static let maxValue = 10.0
+    static let maxValueLineHeight = 0.3
+    static let lineWidth: Double = 330
+    
+    // animation용 variables
+//    static let dataDivisor = Double(maxValue/maxValueLineHeight)
+//    static let adjustedData: [Double] = thisWeek.map({ $0.rate / dataDivisor })
+//    static var animations: [Animation] = []
+    
+    required init?(coder aDecoder: NSCoder) {
+        super.init(node: MacawChartView.createChart(), coder: aDecoder)
+        backgroundColor = .clear
+    }
+    
+    private static func createChart() -> Group {
+        let items: [Node] = addYAxisItems() + addXAxisItems() + createBars()
+        
+        return Group(contents: items, place: .identity)
+    }
+    
+    private static func addYAxisItems() -> [Node] {
+        let maxLines = 5
+        let lineInterval = 2
+        
+        let yAxisHeight: Double = 200
+        let lineSpacing: Double = 40
+        
+        var newNodes: [Node] = []
+        
+        for i in 1...maxLines {
+            let y = yAxisHeight - (Double(i) * lineSpacing)
+            
+            let valueLine = Line(-5, y, lineWidth, y).stroke(fill:Color.gray.with(a: 0.10))
+            let valueText = Text(text: "\(i * lineInterval)", align: .max
+                                 , baseline: .mid, place: .move(-10, y))
+            valueText.fill = Color.gray
+            
+            newNodes.append(valueLine)
+            newNodes.append(valueText)
+        }
+        
+        // y축 선
+//        let yAxis = Line(0, 0, 0, yAxisHeight).stroke(fill:Color.gray.with(a: 0.25))
+//        newNodes.append(yAxis)
+        
+        return newNodes
+    }
+    
+    private static func addXAxisItems() -> [Node] {
+        let chartBaseY: Double = 200
+        var newNodes: [Node] = []
+        
+        for i in 1...thisWeek.count {
+            let x = (Double(i)) * 50
+            let valueText = Text(text: thisWeek[i - 1].weekday, align: .max, baseline: .mid, place: .move(x - 25, chartBaseY + 15))
```

Comment:
```suggestion
            let valueText = Text(
                text: thisWeek[i - 1].weekday,
                align: .max, 
                baseline: .mid,
                place: .move(x - 25, chartBaseY + 15)
            )
```
이건 제 스타일이긴 한데 참고만,,,
한줄로 쭉 쓰는거 보단 위와 같이 작성하는게 가독성에 좀 더 좋은거 같아요